Hosts file not changed (#19832)
kubernetes/website: content/en/docs/tasks/access-application-cluster/ingress-minikube.md

    ```

    This sends requests from hello-world.info to Minikube.
    

>     {{< note >}}If you are running Minikube locally and do not want to edit the hosts file, remove the `host: hello-world.info` entry from the `example-ingress.yaml` file. Instead, browse to the IP address provided by `minikube ip`: 

Reviewer: Let's make it clear that 192.168.99.117 was an example:

```suggestion
    {{< note >}}If you are running Minikube locally and do not want to edit the hosts file, remove the `host: hello-world.info` entry from the `example-ingress.yaml` file. Instead, browse to a URL based on the IP address provided by `minikube ip`. For example, if `minikube ip` outputs `192.168.99.117` then browse to http://192.168.99.117/ {{< /note >}}
```

>     {{< note >}}If you are running Minikube locally and `**do not want to edit the hosts file**`, then remove the entry `host: hello-world.info` from the **example-ingress.yaml** . In this case you should browse to the IP address provided by `minikube ip`. Example : 

Reviewer 2: No need for bold, also one grammar nit.
```suggestion
    {{< note >}}If you are running Minikube locally and do not want to edit the hosts file, remove the `host: hello-world.info` entry from the `example-ingress.yaml` file. Instead, browse to the IP address provided by `minikube ip`: 
```

>     {{< note >}}If you are running Minikube locally and do not want to edit the hosts file, remove the `host: hello-world.info` entry from the `example-ingress.yaml` file. Instead, browse to a URL based on the IP address provided by `minikube ip`. For example, if `minikube ip` outputs `192.168.99.117` then browse to http://192.168.99.117/ {{< /note >}}

Reviewer 3: {{< note >}}If you are running minikube locally and do not want to edit the hosts file,
remove the `host: hello-world.info` entry from the `example-ingress.yaml` file. Instead, browse to a URL based on the IP address provided by `minikube ip`. For example, if `minikube ip` outputs `192.168.99.117` then browse to http://192.168.99.117/ {{< /note >}}

    ```

    {{< note >}}If you are running Minikube locally, you can visit hello-world.info from your browser.{{< /note >}}
    

>     

Reviewer 2: nit: Remove empty line
```suggestion
```
